runtime/vm/ContinuationHelpers.cpp

```diff
@@ -238,11 +243,6 @@ yieldContinuation(J9VMThread *currentThread)
 	 * must be maintained for weakly ordered CPUs, to unsure that once the continuation is again available for GC scan (on potentially remote CPUs), all CPUs see up-to-date stack .
 	 */
 	/* Notify GC of Continuation stack swap */
```

Comment:
This comment was probably for postUnmountContinuation, so be pre-pended to the comment after...